doc/source/release-notes/Percona-XtraDB-Cluster-5.6.34-26.19.rst

```diff
@@ -0,0 +1,102 @@
+.. rn:: 5.6.34-26.19
+
+===================================
+Percona XtraDB Cluster 5.6.34-26.19
+===================================
+
+Percona is glad to announce the release of
+|Percona XtraDB Cluster| 5.6.34-26.19 on December 14, 2016.
+Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/>`_
+or from our :ref:`software repositories <installation>`.
+
+Percona XtraDB Cluster 5.6.34-26.19 is now the current release,
+based on the following:
+
+* `Percona Server 5.6.34-79.1 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.34-79.1.html>`_
+
+* Galera Replication library 3.19
+
+* wsrep API version 26
+
+All Percona software is open-source and free.
+Details of this release can be found in the
+`5.6.34-26.19 milestone on Launchpad
+<https://launchpad.net/percona-xtradb-cluster/+milestone/5.6.34-26.19>`_.
+
+Deprecated
+==========
+
+* The following encryption modes:
+
+  * ``encrypt=1``
+  * ``encrypt=2``
+  * ``encrypt=3``
+
+  The default is ``encrypt=0`` with encryption disabled.
+  The recommended mode now is the new ``encrypt=4``,
+  which uses SSL files generated by MySQL.
+
+  For more information, see :ref:`encrypt-traffic`.
+
+New Features
+============
+
+* PXC-713 Added ``encrypt=4`` mode for SST encryption
+  that uses SSL files generated by MySQL.
+  Modes ``1``, ``2``, and ``3`` are now deprecated.
+
+Fixed Bugs
+==========
+
+* PXC-252 / PXC-426 Optimized IST donor selection logic to avoid SST.
+  Child processes are now cleaned-up and node state is resumed if SST fails.
+
+* PXC-701 Added ``init.ok`` to the list of files that do not get removed during SST.
+
+* PXC-703 Fixed error with ASIO library when building Galera.
```

Comment:
Please add following description. Existing description doesn't tell what has been fixed.
ASIO library fail to ack EPOLLIN event.